src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -175,6 +187,81 @@ func (c *controller) WaitAndUpdateNodesStatus(status *ControllerStatus) {
 	c.log.Infof("Done waiting for all the nodes. Nodes in error status: %d\n", hostsInError)
 }
 
+func (c *controller) hackDNSAddressConflict(wg *sync.WaitGroup) {
+
+	c.log.Infof("Making sure service %s can reserve the .10 address", dnsServiceName)
+
+	defer wg.Done()
+	networks, err := c.kc.GetServiceNetworks()
+	if err != nil || len(networks) == 0 {
+		c.log.Errorf("Failed to get service networks: %s", err)
+		return
+	}
+
+	ip, _, _ := net.ParseCIDR(networks[0])
+	ip4 := ip.To4()
+	if ip4 == nil {
+		c.log.Debugf("Service network is IPv6: %s", ip)
+		return
+	}
+	ip4[3] = 10 // .10 is the conflicting address
+
+	for i := 0; i < 45; i++ {
```

Comment:
45?

Reply:
~15 min (45 * 20 sec) total if nothing fails. The original Ansible hack does 350 (!)
What do you thing the right number would be? To let services start and take an IP.

Reply:
I think the comment is about the random integer in the code.
Consider defining it as const e.g. `fixDNSAddressConflictMaxAttempts = 45`
And adding the comment you wrote above to the code.